numba/targets/linalg.py

```diff
@@ -1004,3 +1004,103 @@ def svd_impl(a, full_matrices=1):
             return (u.T, s, vt.T)
 
         return svd_impl
+
+    @overload(numpy.linalg.qr)
```

Comment:
It seems qr() was added before Numpy 1.8 so you don't need to make the definition conditional.